src/NodeHandler.ts

```diff
@@ -0,0 +1,44 @@
+import { Evented } from '@dojo/core/Evented';
+import { VNodeProperties } from '@dojo/interfaces/vdom';
+import Map from '@dojo/shim/Map';
+
+export enum Type {
```

Comment:
Can we have some documentation on these (and the rest of the module?), is `type` descriptive enough?